contrib/ibm-mq/README.md

```diff
@@ -0,0 +1,116 @@
+## IBM MQ source
+
+This event source is meant to be used as a Container Source within a Knative cluster in order to consume messages from IBM Message Queue.
```

Comment:
nit: is it worth mentioning the implementation detail that it's written as a "container source"? The user of this event source doesn't need to know, or care, do they?

Reply:
Agree, removed "container source" mention